living_documentation_regime/action_inputs.py

```diff
@@ -77,12 +77,8 @@ def get_output_formats() -> list[str]:
         @return: A list of output formats.
         """
         output_formats_json = get_action_input(OUTPUT_FORMATS, "[]").strip().lower()
```

Comment:
Where input is required, no default is defined here in the code.
Check for different values that must be present in validation, as the validation method should be called ASAP in the main process.

Reply:
See the logic here: [7225737](https://github.com/AbsaOSS/living-doc-generator/pull/83/commits/722573738b636828ac16fcc604a640c045a176cb).